app/assets/stylesheets/gradesheet.css.scss

```diff
@@ -296,3 +298,49 @@ input[type="checkbox"]:not(:checked),
 #interaction_badge {
   color: #0882af;
 }
+
+.sorting .up-icon{
+  display: none;
+}
+
+.sorting_asc .up-icon{
+  display: inline-block;
+}
+
+.sorting_desc .up-icon{
+  display: none;
+}
+
+.sorting .down-icon{
+  display: none;
+}
+
+.sorting_desc .down-icon{
+  display: inline-block;
+}
+
+.sorting_asc .down-icon{
+  display: none;
+}
+
+.sorting .sort-icon{
+  display: inline-block;
+}
+
+.sorting_asc .sort-icon{
+  display: none;
+}
+
+.sorting_desc .sort-icon{
```

Comment:
I feel like this chunk of logic could be simplified to

```css
th:not(.sorting_asc) .up-icon {
  display: none;
}

th:not(.sorting_desc) .down-icon {
  display: none;
}

th:not(.sorting) .sort-icon {
  display: none;
}
```

Reply:
did not know about this, thanks for pointing out